src/Command/TestDispatchBookingMessageCommand.php

```diff
@@ -68,16 +70,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     }
 
     /**
-     * @param $email
-     * @param $type
-     * @param $from
-     * @param $to
+     * @param string $email
+     * @param string $type
+     * @param string $from
+     * @param string $to
```

Comment:
I hate this kind of so-called documentation … But you're not to blame – blame society!